skopt/learning/gaussian_process/kernels.py

```diff
@@ -120,7 +130,11 @@ def gradient_x(self, x, X_train):
             sqrt_3_dist = sqrt(3) * dist
             f = np.expand_dims(1 + sqrt_3_dist, axis=1)
 
-            dist_expand = np.expand_dims(sqrt(3) / dist, axis=1)
+            # Gradients when dist equal to zero is zero.
```

Comment:
Wut? Maybe we can find a better way of formulating this (I don't get it at the moment)